### PR TITLE
fix for bz1640164, version 3.10 and 3.11

### PR DESCRIPTION
**Pull request by vwalek**
Fix of https://bugzilla.redhat.com/show_bug.cgi?id=1640164

For version 3.10 and 3.11.
Please, cherry-pick for version 3.10

Thx

### CODE REVIEW

File: install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc [hunk 1]
```diff
@@ -42,13 +42,12 @@ need to adjust these steps to fit your cluster.
...
 
-. Start {product-title} with the updated configuration:
+. Restart {product-title} with the updated configuration:
 +
 ----
-# openshift start \
-    --public-master=https://openshift.example.com:8443 \
-    --master-config=/etc/origin/master/master-config.yaml \
-    --node-config=/etc/origin/node-node1.example.com/node-config.yaml
+/usr/local/bin/master-restart api api
```
Comment (kalexand-rh) [inline install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc]: @vwalek, all commands need to start with $ or #. If you need to run these commands as root, use #.

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL?

File: install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc (hunk 1 above)
Comment (vwalek) [reply install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc]: @kalexand-rh - thx, I did not know that.

Comment (kalexand-rh) [reply install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc]: Happy to help! Thanks for opening the PR.

**Comment by kalexand-rh** [comment]
@stuartchuan, will you PTAL?

**Comment by stuartchuan** [comment]
LGTM

**Comment by kalexand-rh** [comment]
@vwalek, please squash so I can merge.

**Comment by kalexand-rh** [comment]
I'm merging https://github.com/openshift/openshift-docs/pull/14579 instead. Closing.

**Pull request closed without merging**